library/error.c

```diff
@@ -740,11 +672,16 @@ const char * mbedtls_low_level_strerr( int error_code )
             return( "ERROR - This is a bug in the library" );
 #endif /* MBEDTLS_ERROR_C */
 
+#if defined(MBEDTLS_PLATFORM_C)
+        case -(MBEDTLS_ERR_PLATFORM_HW_ACCEL_FAILED):
```

Comment:
Since these error codes are now in error.h, do we still need the `#if defined(MBEDTLS_PLATFORM_C)` ?

Reply:
Good point, but as they were added by the script most probably the #ifdef block will be removed after the change above.

Reply:
Seems that the either the `<generate_errors.pl>` or the `<error code names>` need to be modified to resolve this issue. I'm gonna ask the team.

Reply:
I was not aware that error.c is autogenerated. It's fine with me to leave it as it is.